source/Nevermore.IntegrationTests/QueryableIntegrationFixture.cs

```diff
@@ -669,6 +669,32 @@ public void WhereNotContainsEmpty()
 
             customers.Select(c => c.FirstName).Should().BeEquivalentTo("Alice", "Bob", "Charlie");
         }
+        
+        [Test]
+        public void WhereStringContains()
```

Comment:
We had `WhereNotStringContains` but no`WhereStringContains` so I added it while I was here